eZ/Publish/Core/Search/Legacy/Content/WordIndexer/Repository/SearchIndex.php

```diff
@@ -84,13 +84,11 @@ public function addWords(array $words): void
             ->values(
                 [
                     'word' => ':word',
-                    'object_count' => ':one',
                 ]
-            )
-            ->setParameter(':one', 1, ParameterType::INTEGER);
```

Comment:
This is an unnecessary BC break. You're now forcing maintainers to execute upgrade script (which you need to provide since you've changed schema.yaml). It can be avoided and moreover I don't agree that relying on default values is a safe choice.

Reply:
You might say so if you are looking only from code readability POV. 

Here is excerpt from official insert optimization guidelines:

> Take advantage of the fact that columns have default values. Insert values explicitly only when the value to be inserted differs from the default. This reduces the parsing that MySQL must do and improves the insert speed.

Ref. https://dev.mysql.com/doc/refman/8.0/en/insert-optimization.html

But as mention in PR description: this change have minor impact on overall performance improvement so we can drop this commit.